frontend/src/storage/gradesTable.js

```diff
@@ -0,0 +1,116 @@
+import { percentagesToGrade } from "../utils/constants";
+
+const grades = [
+    {
+        date: '10.04.2022',
```

Comment:
Co do dat to zapytał bym Adriana jak wygląda format dat na backendzie i w bazie. Może np być coś w stylu '2022-04-10'. Jeżeli inny format zapisu daty nie zmieni Twojej koncepcji to okej, ale jak zmieni to dogadajcie się jak ma to wyglądać, żeby później bezboleśnie przechodzić z mocków na endpointy